templates/pkg/resource/references.go.tpl

```diff
@@ -143,6 +143,17 @@ func resolveReferenceFor{{ $field.FieldPathWithUnderscore }}(
 	}
 	return nil
 }
+{{ else if not $isList -}}
```

Comment:
Unless I'm mistaken, `isNested` could be `false` here.... so better to do this?

```
{{ if $isNested and (not $isList) -}}
```

Reply:
It's not necessary, it is always going to be true here because of L132.

Reply:
Indeed, I missed that! :)